alluxio/alluxio_file_system.py

```diff
@@ -1,55 +1,55 @@
+import asyncio
 import hashlib
 import json
 import logging
 import re
 import time
+import weakref
+from dataclasses import dataclass
 from enum import Enum
 
+import aiohttp
 import humanfriendly
 import requests
 from requests.adapters import HTTPAdapter
 
 from .worker_ring import ConsistentHashProvider
+from alluxio.retry import retry_request
+from alluxio.retry import validate_response
 
 logging.basicConfig(
     level=logging.WARN,
     format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
 )
 
 
+@dataclass
 class AlluxioPathStatus:
-    def __init__(
-        self,
-        type,
-        name,
-        path,
-        ufs_path,
-        last_modification_time_ms,
-        human_readable_file_size,
-        length,
-    ):
-        self.type = type
-        self.name = name
-        self.path = path
-        self.ufs_path = ufs_path
-        self.last_modification_time_ms = last_modification_time_ms
-        self.human_readable_file_size = human_readable_file_size
-        self.length = length
+    type: str
+    name: str
+    path: str
+    ufs_path: str
+    last_modification_time_ms: float
+    human_readable_file_size: str
+    length: float
 
 
 class LoadState(Enum):
-    RUNNING = 1
-    VERIFYING = 2
-    STOPPED = 3
-    SUCCEEDED = 4
-    FAILED = 5
+    RUNNING = "RUNNING"
+    VERIFYING = "VERIFYING"
+    STOPPED = "STOPPED"
+    SUCCEEDED = "SUCCEEDED"
+    FAILED = "FAILED"
```

Comment:
can those code refactoring be in the main branch directly?

Reply:
I would just try to merge into dev and test well there before into main